examples/sanity/app/styles/index.css

```diff
@@ -0,0 +1,5 @@
+/* 
```

Comment:
Could you remove irrelevant files like this one? Try to keep examples as minimal as possible (as noted in [the instructions](https://github.com/remix-run/remix/tree/main/examples#contributing)).